core/common/src/main/java/alluxio/exception/ExceptionMessage.java

```diff
@@ -278,6 +278,7 @@
   // job manager
   JOB_DEFINITION_DOES_NOT_EXIST("The job definition for config {0} does not exist"),
   JOB_DOES_NOT_EXIST("The job of id {0} does not exist"),
+  JOB_MASTER_FULL_CAPACITY("Job master is at full capacity"),
```

Comment:
Should we print out the capacity of the job master here?

Reply:
Though if we are to add the value - I don't think we want to add it directly to the message within the exception message class by querying the configuration? Otherwise would we be expecting exception message users to use that as the input value when throwing the exception? I don't think either use sounds ideal

Reply:
I think we can use the `CAPACITY` constant in `JobMaster`